solutions/usaco-784.mdx

```diff
@@ -64,4 +67,39 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+## Python Implementation
+
+```py
+file_in = open('lifeguards.in')
+data = file_in.read().strip().split('\n')
+n = int(data[0])
+
+lifeguards = [list(map(int, data[i + 1].split(' '))) for i in range(n)]
+
+MAX_TIME = 1000
+max_time_covered = 0
```

Comment:
put this between the multiline comment & the for loop

Reply:
Also probably personal preference.